### PR TITLE
remove skip verify from UI.

### PR DESCRIPTION
**Pull request by appleboy**
screen shot 2017-05-05 at 4 08 36 pm

Signed-off-by: Bo-Yi Wu <appleboy.tw@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
   },
   "scripts": {
     "start": "node ./webpack.devserver.js",
-    "build": "NODE_ENV=production webpack",
+    "build": "NODE_ENV=production ./node_modules/.bin/webpack",
```
Comment (tboerger) [inline package.json]: Nope, `npm run` automatically adds `node_modules/.bin` to the path, so please revert that.

Comment (appleboy) [inline package.json]: https://github.com/drone/drone-ui/pull/108/commits/929f5fce7e281e352671284cc24d959453ec89c8 Done

**Comment by bradrydzewski** [comment]
thanks! can you re-submit without the updated go file? I have a routing that auto-generates this :)

**Comment by appleboy** [comment]
@bradrydzewski Done. Please help to review again.

**Comment by tonglil** [comment]
Following up from Gitter, what about removing (or at least hiding) the conceal checkbox as well until it's implementation is completed?

**Pull request merged**